onnxscript/function_libs/torch_aten/graph_building.py

```diff
@@ -65,169 +64,217 @@ def onnx_dtype(self):
+
+    Returns:
+        The unwrapped value.
+    """
     if isinstance(value, dict):
-        value = {
+        return {
             k: v.symbolic_value() if isinstance(v, TorchScriptTensor) else v
             for k, v in value.items()
         }
-    elif isinstance(value, list):
-        value = [v.symbolic_value() if isinstance(v, TorchScriptTensor) else v for v in value]
-    elif isinstance(value, tuple):
-        return tuple(v.symbolic_value() if isinstance(v, TorchScriptTensor) else v for v in value)
-    elif isinstance(value, TorchScriptTensor):
-        value = value.symbolic_value()
-    else:
-        print("value: ", value)
+    if isinstance(value, list):
+        return [v.symbolic_value() if isinstance(v, TorchScriptTensor) else v for v in value]
+    if isinstance(value, tuple):
+        return tuple(
+            v.symbolic_value() if isinstance(v, TorchScriptTensor) else v for v in value
+        )
+    if isinstance(value, TorchScriptTensor):
+        return value.symbolic_value()
     # A normal python value
     return value
 
 
 def _wrap_torch_value_to_tensor(
-    value: Union[Dict[str, Any], List]
-) -> Union[Dict[str, Any], List]:
-    # wrap torch.Value with TorchScriptTensor
+    value: torch.Value
+    | dict[str, torch.Value | str | int | float]
+    | list[torch.Value | str | int | float]
+    | tuple[torch.Value | str | int | float, ...]
+) -> ValidArgumentType | dict[str, ValidArgumentType] | list[ValidArgumentType] | tuple[
+    ValidArgumentType, ...
+]:
+    """Wrap torch.Value to TorchScriptTensor.
+
+    Args:
+        value: The value to wrap.
+
+    Returns:
+        The wrapped value.
+    """
     if isinstance(value, dict):
-        value = {
+        return {
             k: TorchScriptTensor(v) if isinstance(v, torch.Value) else v
             for k, v in value.items()
         }
-    elif isinstance(value, list):
-        value = [TorchScriptTensor(v) if isinstance(v, torch.Value) else v for v in value]
-    elif isinstance(value, tuple):
+    if isinstance(value, list):
+        return [TorchScriptTensor(v) if isinstance(v, torch.Value) else v for v in value]
+    if isinstance(value, tuple):
         return tuple(TorchScriptTensor(v) if isinstance(v, torch.Value) else v for v in value)
-    elif isinstance(value, torch.Value):
-        value = TorchScriptTensor(value)
+    if isinstance(value, torch.Value):
+        return TorchScriptTensor(value)
     return value
 
+
 def _unwrap_tensors_to_torch_values(tensors):
+    # TODO(justinchuby): Do we really need this?
     if isinstance(tensors, Sequence):
         return [_unwrap_tensor_to_torch_value(output) for output in tensors]
     return _unwrap_tensor_to_torch_value(tensors)
 
+
 class TorchScriptEvaluator(evaluator.Evaluator):
+    """An onnxscript Evaluator that captures the graph into torchscript."""
+
     def __init__(self, graph: TorchScriptGraph):
-        self._graph = graph
+        self._graph: TorchScriptGraph = graph
```

Comment:
Looks like redundant

Reply:
That's ok. hopefully it helps type checkers